### PR TITLE
Bump cardano-addresses and fix inspectAddress HRPs

### PR DESCRIPTION
**Pull request by rvl**
### Issue Number

ADP-744

### Overview

- Fixes silly mistake in inspectAddress spending_key_hash_bech32 - input-output-hk/cardano-addresses#117.

### CODE REVIEW

**Review by piotr-iohk — APPROVED** [review]
lgtm.

```
$ curl -X GET http://localhost:8090/v2/addresses/addr1qx244vt9vg0lyld5qnwlkw2gfgt6wteznxngyygvg7f3sd8cc65g386hgcvheanjy99xsvnf0lz47rh6skr20l2dm75qyslaq5 | jq

  "stake_reference": "by value",
  "stake_key_hash_bech32": "stake_vkh1lrr23zyl2arpjl8kwgs556pjd9lu2hcwl2zcdflafh06slsz9nj",
  "stake_key_hash": "f8c6a8889f5746197cf672214a6832697fc55f0efa8586a7fd4ddfa8",
  "spending_key_hash_bech32": "addr_vkh1j4dtzetzrle8mdqymhanjjz2z7nj7g5e56ppzrz8jvvrgx7erh6",
  "address_style": "Shelley",
  "spending_key_hash": "955ab165621ff27db404ddfb39484a17a72f2299a682110c47931834",
  "network_tag": 1
}

```

**Review by paweljakubas — APPROVED** [review]

**Comment by rvl** [comment]
Thanks.

bors r+

File: lib/core-integration/src/Test/Integration/Scenario/API/Shelley/Addresses.hs [hunk 1]
```diff
@@ -356,7 +356,7 @@ spec = describe "SHELLEY_ADDRESSES" $ do
         verify r
             [ expectResponseCode HTTP.status200
             , expectField (Aeson.key "spending_key_hash_bech32" . Aeson._String)
-                (`shouldBe` "addr_test1hwl9tuz8uuqe8cnpv387d5kcj8gyz9r9q30x395vsvue5una0f4")
+                (`shouldBe` "addr_vkht1hwl9tuz8uuqe8cnpv387d5kcj8gyz9r9q30x395vsvue5una0f4")
```
Comment (KtorZ) [inline lib/core-integration/src/Test/Integration/Scenario/API/Shelley/Addresses.hs]:
```
$ bech32 addr_vkh <<< addr_test1hwl9tuz8uuqe8cnpv387d5kcj8gyz9r9q30x395vsvue5una0f4
addr_vkh1hwl9tuz8uuqe8cnpv387d5kcj8gyz9r9q30x395vsvue5e44fh7
```

you cannot actually change the prefix of a bech32-encoded string "just like that" :sweat_smile:, the overall encoding depends on the prefix itself. A different prefix leads to a completely different string. Thus, to change the prefix, one needs to re-encode the whole payload. Our little `bech32` tool handles that just fine.

Comment (rvl) [reply lib/core-integration/src/Test/Integration/Scenario/API/Shelley/Addresses.hs]: Oh yes, thanks for fixing it... I started running this integration test, then forgot to come back to the test result and copy the correct encoding.

**Pull request merged**